build-coordinator/src/main/java/org/jboss/pnc/core/builder/DatastoreAdapter.java

```diff
@@ -48,13 +48,13 @@ public DatastoreAdapter(Datastore datastore) {
         this.datastore = datastore;
     }
 
-    public void storeResult(BuildTask buildTask, BuildResult buildResult) throws DatastoreException {
+    public BuildRecord storeResult(BuildTask buildTask, BuildResult buildResult, int buildRecordId) throws DatastoreException {
```

Comment:
What is the benefit of having a parameter vs. getting the id from task internally?